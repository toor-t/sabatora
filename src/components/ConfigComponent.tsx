//
// ConfigComponent
//
'use strict';
import * as React from 'react';
import Typography from '@material-ui/core/Typography';

export interface IConfigComponentProps {
    // TODO:
    checked?: boolean;
    onChange?: (e: any) => any;
}
const ConfigComponent: React.SFC<IConfigComponentProps> = props => {
    // TODO:
    const { checked, onChange } = props;
    return (
        <div>
            <Typography
                variant="title"
                color="inherit"
                style={{ flexGrow: 1 }}
                noWrap={true}
                align="center"
            >
                設定
            </Typography>
        </div>
    );
};

export default ConfigComponent;
