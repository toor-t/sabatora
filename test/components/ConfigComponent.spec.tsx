import * as React from 'react';
import * as renderer from 'react-test-renderer';

import ConfigComponent from '../../src/components/ConfigComponent';

describe('ConfigComponent', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<ConfigComponent />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
