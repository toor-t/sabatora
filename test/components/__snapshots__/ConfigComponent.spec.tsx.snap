// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConfigComponent renders correctly 1`] = `
<div>
  <h6
    className="MuiTypography-root-1 MuiTypography-h6-18 MuiTypography-colorInherit-30 MuiTypography-noWrap-27 MuiTypography-alignCenter-24"
    style={
      Object {
        "flexGrow": 1,
      }
    }
  >
    設定
  </h6>
</div>
`;
